db/migrate/20240929074158_rename_state_to_status.rb

```diff
@@ -0,0 +1,38 @@
+class RenameStateToStatus < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def up
+    add_column :distributions, :status, :integer
+
+
+    safety_assured do
+      # backfill data from state to status
+      execute <<-SQL
+        UPDATE distributions SET status = state;
+      SQL
+
+      # remove state
+      remove_column :distributions, :state
+
+      change_column_default :distributions, :status, 5
+      change_column_null :distributions, :status, false
```

Comment:
This seems like a way of trying to get around the warning of strong\_migrations. But if anything this doesn't help much. :) Renaming a column really requires something like this: [https://oleg0potapov.medium.com/dangerous-migrations-in-rails-and-how-to-handle-them-77a9f968e36](https://oleg0potapov.medium.com/dangerous-migrations-in-rails-and-how-to-handle-them-77a9f968e36)

Can we just leave the DB column as is and just change the text in the views?

Reply:
@dorner if you are comfortable with having them different going forward, it is obviously a simpler change.

Reply:
How about i create an alias, that way, we can still use status in the code and at the same time we can leave out changes to the db columns.

Do let me know how i should move forward. @cielf

Reply:
I'm leaving that call to @dorner. (I do more the business side, and he makes the calls on technical questions)